e2e/inversify.config.ts

```diff
@@ -31,12 +31,24 @@ import { RightToolbar } from './pageobjects/ide/RightToolbar';
 import { Terminal } from './pageobjects/ide/Terminal';
 import { DebugView } from './pageobjects/ide/DebugView';
 import { WarningDialog } from './pageobjects/ide/WarningDialog';
+import { MultiUserLoginPage } from './pageobjects/login/MultiUserLoginPage';
+import { TestConstants } from './TestConstants';
+import { ILoginPageOcp } from './pageobjects/openshift/ILoginPageOcp';
+import { OpenShiftLoginPage } from './pageobjects/openshift/OpenShiftLoginPage';
+import { OpenShiftConsole4x } from './pageobjects/openshift/OpenShiftConsole4x';
+import { OpenShiftLoginByTempAdmin } from './pageobjects/openshift/OpenShiftLoginByTempAdmin';
 
 const e2eContainer = new Container();
 
 e2eContainer.bind<IDriver>(TYPES.Driver).to(ChromeDriver).inSingletonScope();
-e2eContainer.bind<ILoginPage>(TYPES.LoginPage).to(SingleUserLoginPage).inSingletonScope();
 
+if (TestConstants.TS_SELENIUM_MULTIUSER) {
```

Comment:
Just notation for future, when a count of the special constants for different assemblies will grow up, we should move they to the different module

Reply:
Agree.